backend/src/main/java/gov/cdc/usds/simplereport/api/accountrequest/AccountRequestController.java

```diff
@@ -125,12 +109,54 @@ public void submitAccountRequest(@Valid @RequestBody AccountRequest body) throws
        */
       // send summary email to SR support
       String subject = "New account request";
-      _es.send(sendGridProperties.getAccountRequestRecipient(), subject, body);
+      _es.send(sendGridProperties.getAccountRequestRecipient(), subject, request);
       // send next-steps email to requester
-      _es.sendWithProviderTemplate(body.getEmail(), EmailProviderTemplate.ACCOUNT_REQUEST);
+      _es.sendWithProviderTemplate(request.getEmail(), EmailProviderTemplate.ACCOUNT_REQUEST);
 
-      createAdminUser(org, reqVars);
-      _crm.submitAccountRequestData(body);
+      createAdminUser(
+          request.getFirstName(), request.getLastName(), request.getEmail(), org.getExternalId());
+      _crm.submitAccountRequestData(request);
+    } catch (ResourceException e) {
+      // The `ResourceException` is thrown when an account is requested with an existing org
+      // name. This happens quite frequently and is expected behavior of the current form
+      throw e;
+    } catch (IOException | RuntimeException e) {
+      throw new AccountRequestFailureException(e);
+    }
+  }
+
+  /**
+   * Read the account request and generate an email body, then send with the emailService and create
+   * org without creating a facility
+   */
+  @SuppressWarnings("checkstyle:illegalcatch")
+  @PostMapping("/without-facility-with-emails")
+  @Transactional(readOnly = false)
+  public void submitAccountRequestWithoutFacility(
+      @Valid @RequestBody OrganizationAccountRequest request) throws IOException {
+    try {
+
+      logOrganizationAccountRequest(request);
+
+      Organization org = checkAccountRequestAndCreateOrg(request);
```

Comment:
You may want to run the linter again - the spacing looks a little off to me here. (`./gradlew spotlessApply`, I don't remember if that's covered in our onboarding docs)